tests/typescript_error_recovery/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -37,7 +37,7 @@ class f4 {
   constructor<>() {}
 }
 
-const f5 = function<>() {};
+const f5 = function <>() {};
```

Comment:
It looks ugly.

Reply:
Looks unusual, but generic anonymous functions are used too seldom to worry about their look.

Reply:
I still have to say: PLEASE DO NOT DO THIS.

Reply:
Why feel so strongly about it? Who uses this syntax nowadays anyway?

Reply:
https://github.com/prettier/prettier/pull/3903#issuecomment-554991389

I still doubt "we have consensus that this should be accepted".

> Most users won't come to the issue list before they have problems with `prettier`.

Reply:
This&nbsp;already&nbsp;occurs with&nbsp;the&nbsp;<code>new&nbsp;&lt;T&gt;(…)</code>&nbsp;construct in&nbsp;**TypeScript**.